content/posts/fleek-network-node-healthchecks.md

```diff
@@ -145,47 +153,32 @@ For the ones who followed the [getting started guide](#fleek-network-getting-sta
 We execute a `cURL` request with the `--head` or `-I` flag to show the document info only, in our case the headers of our HTTP response.
 
 ```sh
-curl --head 127.0.0.1:4069
+curl --head 127.0.0.1:80
```

Comment:
```suggestion
curl 127.0.0.1/ping
```

80 is assumed by curl and the ping endpoint will respond with `pong` and code 200